spccore/internal/pathutils.py

```diff
@@ -0,0 +1,28 @@
+import math
+import os
+import re
+import typing
+
+from spccore.internal.timeutils import *
+
+
+# To be compatible with R
+def normalize_path(path: str) -> typing.Union[str, None]:
```

Comment:
`Optional[T]` can be used as as shortcut for `Union[ T, None]`
https://docs.python.org/3/library/typing.html#typing.Optional

Reply:
This is great! Thank you.